src/routes/namespaces/[namespace]/workflows/_workflows-summary-table.svelte

```diff
@@ -1,18 +1,17 @@
-<table class="border-collapse w-full">
-  <thead>
-    <tr class="bg-gray-200">
-      <th class="w-3/12 text-left ">Workflow Name</th>
-      <th class="w-3/12 text-left">Workflow ID</th>
-      <th class="w-1/12">Status</th>
-      <th class="w-2/12 text-right">Started</th>
-      <th class="w-2/12 text-right">Ended</th>
-    </tr>
-  </thead>
-  <slot name="rows" />
-</table>
+<section class="workflow-table border-collapse w-full rounded-lg">
+  <div class="bg-gray-200 flex flex-row p-2">
+    <div class="w-3/12 text-left">Workflow ID</div>
+    <div class="w-3/12 text-left">Type</div>
+    <div class="w-3/12 text-left">Status</div>
+    <div class="w-2/12 text-left">Started</div>
+    <div class="w-2/12 text-left">End</div>
+  </div>
+  <slot />
+</section>
 
 <style lang="postcss">
-  th {
-    @apply bg-gray-200 text-gray-500 text-xs h-6 m-0 p-3 uppercase table-fixed;
+  .workflow-table {
+    display: block;
+    height: calc(100vh - 25%);
```

Comment:
This definitely has some weird implications on the layout. When we scroll down, there is a lot of extra space. We probably are going to want to get rid of this. I'm not going to block this PR over it though.

Now that we have no sidebar and we're about to remove the navigation on the other side, we can probably simplify the layout and remove this later.